pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/LanguageXmlFileMapping.java

```diff
@@ -12,24 +13,24 @@
 	private final static String CATEGORY = "category";
 	private final static String RULESETS = "rulesets";
 
-	private static LanguageXmlFileMapping INSTANCE = new LanguageXmlFileMapping();
```

Comment:
A singleton was problematic for unit tests, could add a #reset method, but then tests couldn't be run in parallel either.